src/datachannel.html

```diff
@@ -47,7 +47,7 @@ <h3>Usage</h3>
         participantAddress = dropDownListElement.value;
 
         // If no particular address is selected, send to all participants.
-        if (participantAddress === 'default') {
+        if (participantAddress === '') {
```

Comment:
await room.sendMessageToParticipant(data, participantAddress);

should suffice I think. Since we pass an empty string (which is falsy) it will broadcast